content/source/docs/enterprise/api/modules.html.md

```diff
@@ -44,13 +44,13 @@ $ curl \
 ```
 
 
-## Publish a Module
+## Publish a Module from a VCS
 
 This endpoint can be used to publish a new module to the registry. The publishing process will fetch all tags in the source repository that look like SemVer versions with optional 'v' prefix. For each version, the tag is cloned and the config parsed to populate module details (input and output variables, readme, submodules, etc.). The [Module Registry Requirements](../../registry/modules/publish.html#requirements) define additional requirements on naming, standard module structure and tags for releases.
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | /registry-modules |
+| Method | Path |
+| :-- | :-- |
+| `POST` | `/registry-modules` |
```

Comment:
Just as a note for future PRs, we do use the extra dashes for readability of the raw Markdown. Template is here: https://raw.githubusercontent.com/hashicorp/terraform-website/master/content/source/docs/enterprise/api/_template.md